File: Contains Duplicate/code.cs
public class Solution {
    public bool ContainsDuplicate(int[] nums) {
            int count = nums.Length;
            if (count == 0 || count == 1)
                return false;
            if (count == 2)
                return nums[0] == nums[1];
            Array.Sort(nums);
            for (int i = 1; i < count; i++)
            {

                if (nums[i] == nums[i - 1])
                    return true;

            }

            return false;
    }
}

File: Word Break/code.java
public class Solution {
    public boolean wordBreak(String s, Set<String> dict) {
        int index = 0;
        int length = dict.size();
        int head = 0;
        boolean found = false;
        String original = s;
        
        StringLengthListSort ss = new StringLengthListSort();
        List<String> list = new ArrayList<String>(dict);
        Collections.sort(list, ss);
        
        for (head = 0; head < length; head++)
        {
            s = original;
            for (int i = head; i < length; i++)
            {
                index = s.indexOf(list.get(i));
                if (index > -1)
                {
                    while (index > -1)
                    {
                        s = s.substring(0, index) +  s.substring(index + (list.get(i)).length());
                        index = s.indexOf(list.get(i));
                    }
                }
            }

            if (s.length() == 0) 
            {
                found = true;
                break;
            }
        }

        return found;
    }
}


class StringLengthListSort implements Comparator<String>{

    @Override
    public int compare(String s1, String s2) {
    return s2.length() - s1.length();
    }
}